Hestia/app/src/main/java/hestia/UI/IpDialog.java

```diff
@@ -46,9 +46,9 @@ public void onClick(View v) {
         switch (v.getId()) {
```

Comment:
we should discuss this switch during our following meeting, it still seems like there should be a better solution.